ccm_web/client/src/models/FeatureUIData.tsx

```diff
@@ -76,5 +77,25 @@ const allFeatures: FeatureUIGroup[] = [
   { id: 'Sections', title: 'Sections', ordinality: 3, features: [mergeSectionCardProps, createSectionsCardProps] }
 ]
 
+const isAuthorizedForRoles = (userRoles: RoleEnum[], requiredRoles: RoleEnum[], featureDescriptionForLogging: string): boolean => {
+  const rolesAllowingAccess = userRoles.filter(requiredRole => requiredRoles.includes(requiredRole))
```

Comment:
I guess it might make more sense to use `userRole` for parameter in the arrow function?

Reply:
Thanks for catching that, I renamed the parameters and missed that.